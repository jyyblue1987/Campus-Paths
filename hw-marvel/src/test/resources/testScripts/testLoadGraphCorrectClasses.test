# Test load graph classes correctly

LoadGraph classes classes.csv

ListNodes classes
ListChildren classes CSE311
ListChildren classes CSE312
ListChildren classes CSE344
ListChildren classes CSE332