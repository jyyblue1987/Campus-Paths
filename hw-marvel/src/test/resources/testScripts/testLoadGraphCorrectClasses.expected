# Test load graph classes correctly

loaded graph classes

classes contains: CSE311 CSE312 CSE331 CSE332 CSE333 CSE344 CSE351 CSE440
the children of CSE311 in classes are: CSE312(A) CSE351(A) CSE440(A)
the children of CSE312 in classes are: CSE311(A) CSE351(A) CSE440(A)
the children of CSE344 in classes are:
the children of CSE332 in classes are: CSE331(C) CSE333(C)